src/Symfony/Bridge/Twig/Extension/SecurityExtension.php

```diff
@@ -41,6 +41,8 @@ public function isGranted(mixed $role, mixed $object = null, ?string $field = nu
         }
 
         if (null !== $field) {
+            trigger_deprecation('symfony/twig-bridge', '7.3', 'Passing the "$field" argument to the "is_granted" Twig function is deprecated and will be removed in version 8.0.');
```

Comment:
```suggestion
            trigger_deprecation('symfony/twig-bridge', '7.3', 'The "$field" parameter of the "is_granted" Twig function is deprecated and will be removed in version 8.0.');
```
Even more correct I think